src/uu/ls/src/ls.rs

```diff
@@ -2245,52 +2230,73 @@ fn enter_directory(
 
     display_items(&entries, config, state, dired)?;
 
+    let iter = entries
```

Comment:
iter isn't the best name 
please call it what it is doing / used for instead of the type of data